service/src/main/java/bio/terra/workspace/service/resource/controlled/ControlledResourceService.java

```diff
@@ -222,6 +222,7 @@ public String cloneGcsBucket(
             .operationType(OperationType.CLONE)
             .addParameter(ControlledResourceKeys.DESTINATION_WORKSPACE_ID, destinationWorkspaceId)
             .addParameter(ControlledResourceKeys.DESTINATION_RESOURCE_ID, destinationResourceId)
+            .addParameter(ControlledResourceKeys.DESTINATION_FOLDER_ID, null)
```

Comment:
This looks strange to me. If resource has folderId, it's weird we're setting folder id to null.

Could we:
- Delete 225 and 342
- In a later step, only set DESTINATION_FOLDER_ID if resource has folder id
- FYI in CloneAllFoldersStep, `inputParameters.get(ControlledResourceKeys.DESTINATION_FOLDER_ID, String.class)` will return null if that key isn't set.